libcodechecker/analyze/analyzer.py

```diff
@@ -80,6 +82,22 @@ def perform_analysis(args, context, actions, metadata):
     analyzers, _ = analyzer_types.check_supported_analyzers(
         analyzers, context)
 
+    ctu_collect = False
+    ctu_analyze = False
+    ctu_dir = ''
+    ctu_func_map_cmd = ''
+    # ctu_in_memory = False
+    if hasattr(args, 'ctu_phases'):
```

Comment:
While `hasattr` seems to be correct, we are using `'dest' in args` for these checks.